lib/source-destination/source-destination.ts

```diff
@@ -261,8 +258,10 @@ export class SparseStreamVerifier extends Verifier {
 			});
 			transform.once('error', (error: Error) => {
 				originalStream.unpipe(transform);
+				// eslint-disable-next-line
```

Comment:
balena-lint expects us to replace `@ts-ignore`s with `@ts-expect-error`s with a description, not silence them.
Even if the description is `to make TS happy/ I don't know why` it's way more helpful, since if we later fix the typings, TS will let us know that the `@ts-expect-error` is no longer required.
Applies in more places afaict.